### PR TITLE
Adding role information to SRT Procedures.

### PR DESCRIPTION
**Pull request by albertcrowley**
Added explicit SRT team roles and mapping to cloud.gov user configuration to 
fulfill the NIST-800-53 AC-5  (Separation of Duties) and AC-6 (Least Privilege) controls.

### CODE REVIEW

**Comment by csmcallister** [comment]
Looks good. Thanks for the add. Do you want me to merge this or do you think there are going to be more updates that you'd like to commit to this branch?

**Comment by albertcrowley** [comment]
I'm happy to keep this open until we submit the SSP.

File: docs/procedures.md [hunk 1]
```diff
@@ -77,3 +82,35 @@ We rely on cloud.gov’s [audit tool](https://logs.fr.cloud.gov).  In the event
 
 ## Unsupported System Components Policy
 At present, there are no unsupported components in the SRT system. If currently-used components become deprecated and/or unsupported, we will bump their versions to the most recent stable release. If we're unable to make those bumps, then we will provide justification and document approval for the continued use of the unsupported system components within the relevant GitHub issue.
+
+## GSA Solutions Life Cycle (SLC)
```
Comment (albertcrowley) [inline docs/procedures.md]: Did this come from some existing doc or policy? It's very waterfall-like which probably doesn't matter much in the end.

Comment (csmcallister) [reply docs/procedures.md]: I agree. It comes from this:  https://docs.google.com/document/d/1nWx5hPQNF9j31t_vI8yNHGcths_Q-Gs2UkRfZGBdwCM/edit#

To satisfy the control, I took the list of required phases and briefly discussed how we'd perform the activities they list for each one.

The doc recommends a variety of non-waterfally "strategies" that a team can use to move through these phases, with agile being one of them. But the SSP doesn't ask us to declare our strategy, so I figured I'd keep that out of the docs.

**Pull request merged**